### ISSUE TITLE
Fix solr installation instructions, in the dev. and installation guide

### ISSUE THREAD

**Issue by landreev** [IQSS/dataverse#4748]
Update: the issue below is a non-issue. The instruction in the develop branch actually says "install solr 7.3". Closing the issue. 

The installation instructions cannot be followed as they are in the guide now. It fails in a way that is super confusing to the user. 
Specifically, the following curl command: 
```
curl -O https://raw.githubusercontent.com/IQSS/dataverse/develop/conf/solr/7.2.1/schema.xml
```
(and the similar line for solrconfig.xml). 
The "7.2.1" does not exist - the only schema we provide is .../conf/solr/7.3/schema.xml. However, due to the way github works, the curl command above quietly saves the file "schema.xml" - with the message "404 NOT FOUND" inside - making the user think they have downloaded the schema. Which they then copy into their solr config, making solr all sorts of unhappy - and in a very confusing way.

The above is from the dev. guide. But a similar problem exists ins the "normal" installation instructions: We are telling the users to install version 7.2.1; but then instructing them to install the schema and solrconfig that we supply - which are 7.3-specific. 

A quickest fix for the dev. installer version would be not to touch the documentation at all, but to put the conf/solr/7.2.1 back in place.

**Comment by djbrooke** [comment]
@landreev I'm not sure I understand the proposed fix - is it to drop back to 7.2.1 (implemented in #4158) instead of 7.3 (implemented in #4579)? We moved to 7.3 because we had not cut the branch yet and because of some security updates that were included in it.

**Comment by landreev** [comment]
@djbrooke I was only suggesting to keep supplying the schema for the version 7.2.1 as well, in addition to 7.3. So if anyone is still trying to install 7.2.1, it would work for them as well. Dataverse appears to be working just fine with 7.2.1 (dvn-build still has it). 
But it all appears to be moot point - it really looks like the instructions are perfectly fine ("install solr 7.3") in the latest branch. I was looking at some older version of the instruction yesterday, because of some local issues. 
So I'm closing this issue.
